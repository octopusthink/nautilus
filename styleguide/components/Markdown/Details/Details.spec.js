import React from 'react';

import { Details, DetailsSummary } from './index';

describe('Markdown Details', () => {
	it('should render a Details', () => {
		const actual = render(
			<Details>
				<DetailsSummary>Solution</DetailsSummary>
				This is a hidden text.
			</Details>
		);

		expect(actual).toMatchSnapshot();
	});
});
