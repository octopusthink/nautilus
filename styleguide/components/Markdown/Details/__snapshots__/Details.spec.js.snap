// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markdown Details should render a Details 1`] = `
<details
  class="rsg--details-0"
>
  <summary
    class="rsg--summary-2"
  >
    Solution
  </summary>
  This is a hidden text.
</details>
`;
